README.md

```diff
@@ -65,11 +65,13 @@ Options:
       --log-teamcity=LOG-TEAMCITY                Log test execution in Teamcity format to file.
   -m, --max-batch-size=MAX-BATCH-SIZE            Max batch size (only for functional mode). [default: 0]
       --no-test-tokens                           Disable TEST_TOKEN environment variables. (default: variable is set)
+      --order-by[=ORDER-BY]                      Run tests in order: default|random|reverse // this param doesn't pass order-by to phpunit, use passthru
```

Comment:
Maybe we should instead pass this also to the underlying PHPUnit command, also with seed for `random`, otherwise only the TestCases are random but within the same TestCase the tests keep running in default order, WDYT?

Reply:
Idk really, I one case I think its more clear that tools with random order pass this to phpunit
Otherwise I think maybe not all people want this, may be we can pass all this to phpunit, but add extra cmd line parameter to disable this behaviour

Reply:
And if we want to passthru orderby and seed, pls help me with the place in code (idk how to do it)

Reply:
Added to filtered array, this will pass params  to phpunit

Reply:
now everything ok except windows 8, idk how to fix this, and where is real problem

Reply:
okey, re-push helped and now everything is okey